sr_robot_commander/src/sr_robot_commander/sr_hand_commander.py

```diff
@@ -29,13 +29,14 @@ class SrHandCommander(SrRobotCommander):
 
     __set_force_srv = {}
 
-    def __init__(self, name="right_hand"):
+    def __init__(self, name="right_hand", prefix="rh"):
```

Comment:
bit of a duplicated information here but I wasn't sure how to get rid of it. We need to look carefully at our launch files and harmonize everything.